src/components/entity/ha-state-label-badge.ts

```diff
@@ -110,7 +110,9 @@ export class HaStateLabelBadge extends LitElement {
         return null;
       case "sensor":
       default:
-        return state.state === UNKNOWN
+        return state?.attributes.device_class === "moon__phase"
+          ? null
+          : state.state === UNKNOWN
```

Comment:
Why in `default` the `moon__phase` device class is of the sensor domain I guess?

Reply:
Because sensor will fall to the default logic otherwise.